plugins/README.md

```diff
@@ -31,15 +31,18 @@ docker run -it -v $(pwd):/mnt \
 Tips:
 
 -   When benchmarking and publishing, compile a release (optimized) wasm build.
--   To see a trace of logs and wasm ABI calls, add `--loglevel=TRACE`.
 -   Try sending empty or invalid input. Verify your plugin doesn't crash.
+-   To see plugin-emitted logs on the console, add `--logfile=/dev/stdout`.
```

Comment:
I think we should specify that the complete flag is --test_arg=--loglevel=TRACE.

Reply:
That's how you set flags under `bazel test`. However most users will be interacting with the tool directly, so `--loglevel` is the flag they'd use.